code/modules/mob/living/carbon/human/npc/dungeon_pack_npc.dm

```diff
@@ -0,0 +1,768 @@
+		H.cast(null, src)
+
+/datum/outfit/job/roguetown/npc/dungeon_paladin
+	name = "Dungeon Paladin"
+
+	pre_equip(mob/living/carbon/human/H)
+		..()
+		head = /obj/item/clothing/head/roguetown/helmet/skullcap
+		mask = /obj/item/clothing/mask/rogue/facemask
+		armor = /obj/item/clothing/cloak/tabard/crusader/tief
+		shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/heavy
+		armor = /obj/item/clothing/suit/roguetown/armor/chainmail/hauberk
+		neck = /obj/item/clothing/neck/roguetown/gorget
+		pants = /obj/item/clothing/under/roguetown/chainlegs
+		shirt = /obj/item/clothing/suit/roguetown/armor/chainmail/iron
+		armor = /obj/item/clothing/cloak/tabard/crusader/tief
+		neck = /obj/item/clothing/neck/roguetown/coif
+		gloves = /obj/item/clothing/gloves/roguetown/chain/iron
+		pants = /obj/item/clothing/under/roguetown/chainlegs
+		wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+		shoes = /obj/item/clothing/shoes/roguetown/armor
+		r_hand = /obj/item/rogueweapon/sword
+		l_hand = /obj/item/rogueweapon/shield/wood
+
+		H.STASTR = 15
+		H.STASPD = 15
+		H.STACON = 15
+		H.STAEND = 14
+		H.STAINT = 8
+		H.STAPER = 17
+
+// 3. Cleric
+/mob/living/carbon/human/species/human/northern/dungeon_base/cleric
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/cleric/after_creation()
+	..()
+	equipOutfit(new /datum/outfit/job/roguetown/npc/dungeon_cleric)
+	possible_rmb_intents += /datum/rmb_intent/swift
+
+/datum/outfit/job/roguetown/npc/dungeon_cleric
+	name = "Dungeon Cleric"
+
+	pre_equip(mob/living/carbon/human/H)
+
+		..()
+		mask = /obj/item/clothing/mask/rogue/facemask
+		head = /obj/item/clothing/head/roguetown/necrahood
+		shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/heavy
+		armor = /obj/item/clothing/suit/roguetown/shirt/robe/necra
+		neck = /obj/item/clothing/neck/roguetown/gorget
+		gloves = /obj/item/clothing/gloves/roguetown/leather/advanced
+		wrists = /obj/item/clothing/wrists/roguetown/bracers/leather/advanced
+		shoes = /obj/item/clothing/shoes/roguetown/armor/leather/advanced
+		r_hand = /obj/item/rogueweapon/mace/wsword
+
+		H.STASTR = 12
+		H.STASPD = 10
+		H.STACON = 12
+		H.STAEND = 12
+		H.STAINT = 11
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/cleric/use_combat_abilities()
+	if(src.stat != CONSCIOUS || world.time < next_cast)
+		return
+
+	next_cast = world.time + 100  // 10 секунд
+
+	var/roll = rand(1, 2)
+
+	if(roll == 1)
+		// === Churn Undead ===
+		var/list/targets = list()
+		for(var/mob/living/L in view(8, src))
+			if(L != src && L.stat != DEAD && (!L.faction || disjoint_lists(L.faction, src.faction)))
+				targets += L
+
+		if(targets.len)
+			src.say("ZIZO REBUKES!!")
+			var/obj/effect/proc_holder/spell/targeted/churnnpc/C = new(src)
+			C.cast(targets, src)
+
+	else if(roll == 2)
+		// === Lesser Heal ===
+		src.say("Blessings upon our wounded!")
+		var/obj/effect/proc_holder/spell/targeted/lesser_heal_npc/H = new(src)
+		H.cast(null, src)
```

Comment:
the explicit roll var and number checks are unnecessary, it's just equivalent to prob(50).
```suggestion
	if(prob(50))
		// === Churn Undead ===
		var/list/targets = list()
		for(var/mob/living/L in view(8, src))
			if(L != src && L.stat != DEAD && (!L.faction || disjoint_lists(L.faction, src.faction)))
				targets += L

		if(targets.len)
			src.say("ZIZO REBUKES!!")
			var/obj/effect/proc_holder/spell/targeted/churnnpc/C = new(src)
			C.cast(targets, src)

	else
		// === Lesser Heal ===
		src.say("Blessings upon our wounded!")
		var/obj/effect/proc_holder/spell/targeted/lesser_heal_npc/H = new(src)
		H.cast(null, src)
```